api/src/main/java/com/epam/pipeline/manager/preference/SystemPreferences.java

```diff
@@ -947,6 +947,9 @@ public class SystemPreferences {
     public static final IntPreference SEARCH_AGGS_MAX_COUNT = new IntPreference("search.aggs.max.count",
             20, SEARCH_GROUP, pass);
 
+    public static final IntPreference SEARCH_EXPORT_PAGE_SIZE = new IntPreference("search.export.page.size",
```

Comment:
This export feature related to faceted search so it is better to move it to `FACETED_FILTER_GROUP` and name appropriately